src/transformers/models/detr/feature_extraction_detr.py

```diff
@@ -1275,12 +1275,13 @@ def post_process_instance_segmentation(
             # Get segmentation map and segment information of batch item
             target_size = target_sizes[i] if target_sizes is not None else None
             segmentation, segments = compute_segments(
-                mask_probs_item,
-                pred_scores_item,
-                pred_labels_item,
-                mask_threshold,
-                overlap_mask_area_threshold,
-                target_size,
+                mask_probs=mask_probs_item,
+                pred_scores=pred_scores_item,
+                pred_labels=pred_labels_item,
+                mask_threshold=mask_threshold,
+                overlap_mask_area_threshold=overlap_mask_area_threshold,
+                label_ids_to_fuse=[],
```

Comment:
@alaradirik While testing `instance` subtask, I found that the returned masks did not respect `target_size`.
I pinpointed that this call was sending `target_size` to `ids_to_fuse` instead of target_size.

`post_process_instance_segmentation` doesn't take a `label_ids_to_fuse` as a parameter, is that expected ?

Future note:

We can write functions as :
```python
def myfunction(*, arg1, arg2):
   pass
```
And this forbids the caller from calling `myfunction(a, b)` he is forced to use `myfunction(arg1=a, arg2=b)` helping prevent this sort of issue.
I don't think we have to do it, just saying that it's a handy tool.

Reply:
Thanks for the `target_size` fix @Narsil! And yes, `post_process_instance_segmentation` doesn't and shouldn't take a `label_ids_to_fuse` as a parameter.

I'm more in favor of using named arguments but both are fine with me really.